pull_request_approval_policy.md

```diff
@@ -0,0 +1,7 @@
+# Pull Request Approval Policies
+
+- CI/CD is required to pass
+- 2 Reviewers are required for all Pull Requests
+- All Change Requests must be resolved prior to merging
+- For CI/CD that deploy to a production system schedule a maintenance event if
```

Comment:
I think this last statement is a little wishy-washy.  How about:

- Changes that impact service availability must only be merged during a pre-scheduled maintenance window.

Reply:
updated.